playwright_dotnet_framework.BusinessLayer/Models/GetWidgetByIdModels/Content.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace playwright.dotnet.framework.BusinessLayer.Models.GetWidgetByIdModels
+{
+    public class Content
+    {
+        public List<Result> Result { get; set; }
```

Comment:
this should be declared as `nullable` and the unneeded System imports should be removed